Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -38,6 +38,10 @@ static CClientExplosionManager*  m_pExplosionManager;
 static CClientProjectileManager* m_pProjectileManager;
 static CClientSoundManager*      m_pSoundManager;
 
+static CLuaPhysicsRigidBodyManager*       m_pLuaPhysicsRigidBodyManager;
```

Comment:
Variables must always be initialized before use

Reply:
is is, at 83-85 line in the same way as statics above